debian/postinst.d/14-python-packages.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+main() {
+    wget https://bootstrap.pypa.io/get-pip.py && \
```

Comment:
python3 and pip should be a dependency to the deb package.

Reply:
if you use wget, it needs to be a dependency as well.
but I prefer to put pip as a dependency to the deb package

Reply:
45df82a409fd2371d6f5a100dab8c9d96704182d